docs/api/api.njk

```diff
@@ -1,12 +1,29 @@
----
+--- 
 pagination: 
-    data: apis.functions
-    size: 1
-    alias: api
-permalink: "api/{{ api.name }}/"
-layout: api.njk
+    data: apis.functions 
+    size: 1 
+    alias: api 
+permalink: "api/{{ api.name }}/" 
+layout: api.njk 
 ---
 
+{% macro simpleTypeParser(type) -%}
```

Comment:
can we call this macro 
```suggestion
{% macro linkToType(type) -%}
```
?

Reply:
Yes, I will refactor the code and submit a new commit.